conf/application.conf

```diff
@@ -161,7 +161,7 @@ assets {
 urls {
   login = "http://localhost:9949/auth-login-stub/gg-sign-in"
   loginContinue = "http://localhost:8503/plastic-packaging-tax/registration"
-  incorpIdCallback = "http://localhost:8503/plastic-packaging-tax/incorp-id-callback"
+  grsCallback = "http://localhost:8503/plastic-packaging-tax/grs-callback"
```

Comment:
Not part of your change, but why do we need to define our own end-point in config?  We could use Routes.

Reply:
I suspect it's because we need a fully qualified URL rather than a relative one.

Reply:
AppConfig could still generate the full url with the `platform.frontend.host` config value.  But it doesn't matter.